src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -63,38 +71,25 @@ public int getSize() {
     }
 
     private void resize() {
-        size = 0;
-        threshold *= (int) (LOAD_FACTOR * table.length);
-        Node<K, V>[] oldNode = table;
-        table = new Node[table.length * 2];
-        for (Node<K, V> node : oldNode) {
-            while (node != null) {
-                put(node.key, node.value);
-                node = node.nextNode;
+        if (size > threshold) {
```

Comment:
I think **size == threshold** would be more correct because in method put after **resize** we often do **size++** and  in this code, map has already 13 elements and then we resize it